token/program/src/state.rs

```diff
@@ -46,6 +46,9 @@ pub struct Account {
     pub delegated_amount: u64,
     /// Optional authority to close the account.
     pub close_authority: COption<Pubkey>,
+    /// An Account is required to be rent-exempt. This value logs the reserve required to be
+    /// rent-exempt so that wrapped SOL accounts do not drop below this threshold.
+    pub rent_exempt_reserve: u64,
```

Comment:
Dare we combine this with `is_native`?

`pub is_native: COption<u64>`?  Where the held u64 is the reserve?

Reply:
Ooh, I like it. Same # of bytes, but makes it more explicit that the reserve only affects wrapped SOL accounts